### DISCUSSION TITLE
bash: line 1: nerdctl: command not found

### DISCUSSION THREAD

**Discussion by christhomas** [lima-vm/lima#729] [Q&A] [answered]
lima version: 0.9.0
machine: m1 max macbook

### Description

When I install lima from brew using the instructions on the README, I get so far that I can see the virtual machine running. But when I try to run `lima nerdctl` I get `bash: line 1: nerdctl: command not found`

So I'm trying to hunt down the missing instructions. But I've not found out what it is yet. Does anybody know?

**Comment by afbjorklund** [comment, marked as answer]
This seems to be due to using the new `--name=default template://docker` parameters, which can be confusing.

* #730

Needs to use `lima docker` with it, not `lima nerdctl`.

Previously the docker template instance was called docker, so less chance of being confused with nerdctl default.
